src/components/ContentHighlights/tests/ContentHighlightSetCard.test.jsx

```diff
@@ -0,0 +1,60 @@
+import { screen, render, fireEvent } from '@testing-library/react';
+import { MemoryRouter } from 'react-router';
+import '@testing-library/jest-dom/extend-expect';
+import { Provider } from 'react-redux';
+import configureMockStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+import ContentHighlightSetCard from '../ContentHighlightSetCard';
+
+const mockStore = configureMockStore([thunk]);
+
+const mockData = {
+  title: 'Test Title',
+  highlightUUID: 'test-uuid',
+  enterpriseSlug: 'test-enterprise-slug',
+};
+
+const publishedData = {
+  ...mockData,
+  isPublished: true,
+};
+
+const unpublishedData = {
+  ...mockData,
+  isPublished: false,
+};
+
+const initialState = {
+  portalConfiguration: {
+    enterpriseSlug: 'test-enterprise-id',
+  },
+  highlightUUID: 'test-uuid',
+};
+
+const ContentHighlightSetCardWrapper = (props) => (
+  <MemoryRouter>
```

Comment:
nit: check out the `renderWithRouter` test utility function. an older version of it still exists in this repo actually in `./src/components/test/testUtils.jsx`. It also exists in `@edx/frontend-enterprise-utils` [here](https://github.com/openedx/frontend-enterprise/blob/master/packages/utils/src/test-utils.jsx#L7)